introduction.md

```diff
@@ -0,0 +1,5 @@
+Go away, I don't want any,
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.